multinet/types.py

```diff
@@ -5,16 +5,15 @@
 EdgeDirection = Literal["all", "incoming", "outgoing"]
 TableType = Literal["all", "node", "edge"]
 
-WorkspacePermissions = TypedDict(
-    "WorkspacePermissions",
-    {
-        "owner": str,  # this is the `sub` property of the `User` dataclass
-        "maintainers": List[str],
-        "writers": List[str],
-        "readers": List[str],
-        "public": bool,
-    },
-)
+
+class WorkspacePermissions(TypedDict):
+    """Permissions on a Workspace."""
+
+    owner: str  # this is the `sub` property of the `User` dataclass
```

Comment:
I wonder if we should just declare a type alias, `UserSub` that equals `str`. Just something to think about. Maybe it's more trouble than it's worth.

Reply:
Perhaps, but I think the with the new ORM changes it would become moot.